tests/ci/reporter_tests.js

```diff
@@ -221,6 +224,48 @@ describe('test reporters', function() {
             '# tests 3',
             '# pass  1',
             '# skip  1',
+            '# todo  0',
+            '# fail  1',
+            ''
+          ]);
+        });
+      });
+
+      context('with todos', function() {
+        it('writes out TAP with failure info', function() {
```

Comment:
This test is failing for me when checking out the branch locally:
```
 1) test reporters tap reporter with error-only output with todos writes out TAP with failure info:

      AssertionError: expected [ Array(16) ] to deeply equal [ Array(22) ]
      + expected - actual

         "        browser log: |"
         "            I am a log"
         "            Useful information"
         "    ..."
      +  "not ok 2 phantomjs - [5 ms] - it is a passing todo"
      +  "    ---"
      +  "        browser log: |"
      +  "            I am a log"
      +  "            Useful information"
      +  "    ..."
         ""
         "1..2"
         "# tests 2"
         "# pass  0"

      at Context.<anonymous> (tests/ci/reporter_tests.js:400:18)
      at processImmediate (internal/timers.js:439:21)
```

Mind taking a look?

Reply:
I believe I've fixed the issue.

Reply:
Thanks for the fix!